src/components/searchBar.jsx

```diff
@@ -3,9 +3,9 @@ import SearchIcon from '@mui/icons-material/Search';
 
 export default function SearchBar() {
     return (
-        <form method='get' action="/" style={{ marginTop: '16px' }}>
+        <form method='get' action="/tv-shows" style={{ marginTop: '16px' }}>
```

Comment:
Just a small suggestion: consider using `/search` instead of `/tv-show` for the URL. It's a more conventional and user-friendly approach that enhances the user experience and aligns with industry best practices. Same thing for components/function names.